utils.go

```diff
@@ -189,8 +189,9 @@ func CellsToString(cells []Cell) string {
 func TrimCells(cells []Cell, w int) []Cell {
 	s := CellsToString(cells)
 	s = TrimString(s, w)
+	runes := []rune(s)
 	newCells := []Cell{}
-	for i, r := range s {
+	for i, r := range runes {
```

Comment:
Does this fix an issue you were having? Iterating through a string (as before) should give you runes already.

Reply:
As I said at #226, when Iterating a Chinese string the value is rune no problem, but index is count as byte. So it will be 0, 3, 6, 9……So there will be an 'index out of range'.

Reply:
Oh interesting, thanks for explaining that.